app/models/operatingsystems/Vsphere.rb

```diff
@@ -0,0 +1,23 @@
+class Vsphere < Operatingsystem
+  PXEFILES = {:kernel => "mboot.c32", :bootcfg => "boot.cfg", :initrd => ""}
+
+  class << self
+    delegate :model_name, :to => :superclass
+  end
+
+  def mediumpath(host)
+	  medium_uri(host).to_s
```

Comment:
Nitpick, too much whitespace at the start of this line - it should have four spaces.